src/coreclr/jit/gentree.cpp

```diff
@@ -380,6 +380,143 @@ bool GenTree::IsNodeProperlySized() const
 }
 #endif
 
+#ifdef DEBUG
+//------------------------------------------------------------------------
+// Dump: Print a textual representation of this ILLocation.
+//
+// Notes:
+//    For invalid ILLocations, we print '???'.
+//    Otherwise the offset and flags are printed in the format 0xabc[EC].
+//
+void ILLocation::Dump() const
```

Comment:
I guess if you move the class definitions to a new header these should move to a new .cpp file?